### ISSUE TITLE
Create Organization Events Page + Backend

### ISSUE THREAD

**Issue by hanmosh** [hack4impact-calpoly/ecologistics-shared-calendar#90]
PREREQ: #78  -> if this is not done yet, just build the page and connect what you can

Use existing components to create the organization events page as shown in the Figma: 

DONT ADD POSTPONED EVENTS FOR NOW 

<img width="495" alt="Screenshot 2024-05-01 at 7 19 50 PM" src="https://github.com/hack4impact-calpoly/ecologistics-shared-calendar/assets/113932134/ff13db0b-54bb-4111-9e98-777c1b1f8e85">

Connect all the fields to the backend: 
- when a user makes a request for an event from the calendar it shows up in the "pending", once the event is approved or denied it moves to the correct tables 
- when user deletes an event it actually removes from the database (or at least from the calendar and event bar)
- if an event time is in the past, it is in the past events table 
- also add a place for notes in declined events -> think about how we might want to store this in the backend
